client/components/mma/delivery/address/DeliveryAddressForm.tsx

```diff
@@ -433,6 +434,67 @@ export const DeliveryAddressUpdate = (props: WithProductType<ProductType>) => {
 		};
 	`;
 
+	const hasNationalDelivery = Object.values(
+		contactIdToArrayOfProductDetailAndProductType,
+	)
+		.flatMap(flattenEquivalent)
+		.some(({ productDetail }) => {
+			return GROUPED_PRODUCT_TYPES.subscriptions
+				.mapGroupedToSpecific(productDetail)
+				.productType.includes('nationaldelivery');
```

Comment:
```suggestion
				.productType === 'nationaldelivery';
```
we can compare the entire string, it's not looking for a substring